examples/gno.land/r/demo/userbook/userbook_test.gno

```diff
@@ -0,0 +1,81 @@
+package userbook
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/testutils"
+	"gno.land/p/demo/ufmt"
+)
+
+func TestGetSignupsInRange(t *testing.T) {
+	// Sign up 20 users + deployer
+	for i := 0; i < 20; i++ {
+		addrName := ufmt.Sprintf("test%d", i)
+		caller := testutils.TestAddress(addrName)
+		std.TestSetOrigCaller(caller)
+		SignUp()
+	}
+
+	testCases := []struct {
+		name                  string
+		page                  int
+		pageSize              int
+		expectedNumberOfUsers int
+	}{}
+
+	for _, tc := range testCases {
+		t.Run("", func(t *testing.T) {
+		})
+	}
```

Comment:
🤷‍♂️ 
```suggestion
```

Reply:
Working on in now!

Reply:
[8ca5956](https://github.com/gnolang/gno/pull/1949/commits/8ca5956cfffc06dbe0f6cb6c4b77bc86220d48e6)!